hs_core/views/resource_folder_hierarchy.py

```diff
@@ -172,7 +172,9 @@ def data_store_folder_unzip(request):
     try:
         unzip_file(user, res_id, zip_with_rel_path, bool_remove_original)
     except SessionException as ex:
-        return HttpResponse(ex.stderr, status=500)
+        specific_msg = "We don't allow unzipping to override existing files. " \

```

Comment:
I like the original message better, which is ```We don't allow unzipping to override existing files. Please check to make sure this is not the case - Error: not-so-informative iRODS message.``` @dtarb Can you check this message and revise it as needed?